src/lib/dependencies.ts

```diff
@@ -65,9 +75,14 @@ export function resolveAllDependencies (options: Options): Promise<DependencyTre
  */
 export function resolveDependency (dependency: Dependency, options: Options): Promise<DependencyTree> {
   const { type, location, raw, meta } = dependency
-
   if (type === 'registry') {
+    // DEBUGGING
+    console.log('resolveDependency starts', location, raw)
```

Comment:
Remember to remove debugging later. If you want to put it behind a flag, use the `debug` module.